bigchaindb-server/bigchaindb/consensus.py

```diff
@@ -18,7 +18,7 @@ def validate_transaction(bigchain, transaction):
         for documentation.
 
         """
-        return transaction.validate(bigchain)
+        return transaction.validate(bigchain) and validate_asset(transaction, bigchain)
```

Comment:
Can we please split the return and log something here?
This will make it easy to know if the `transaction.validate(bigchain)` or `validate_asset(transaction, bigchain)` failed.